sample/src/main/java/com/example/api/GetConfigurationResponse.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.api
+
```

Comment:
Everything in the `com.example.api` package is not used yet. It'll be used later when I add V1 and V2 examples